src/abhishekSJun20/ExtractItemId.java

```diff
@@ -0,0 +1,39 @@
+package abhishekSJun20;
+/*Extract ItemId from below input
+
+String input = https://www.facebook.com/technocredits/temId=12345/
+String input = https://www.facebook.com/technocredits/empid=123&itemId=12345/
+String input = https://www.facebook.com/technocredits/itemId=12345&empid=123/
+String input = https://www.facebook.com/technocredits/empid=123/
+Output : 
+item id is 12345 
+        OR
+no item id present.*/
+
+public class ExtractItemId {
+	
+	void displayItemID(String input) {
+		String itemId = "";
+		if (input.contains("itemId")) {
+			String temp = input.substring(input.indexOf("itemId"));
+			for (int index = 0; index < temp.length(); index++) {
+				if (temp.charAt(index) >='0' && temp.charAt(index) <='9') {
+					itemId += temp.charAt(index);
+					if (!(temp.charAt(index + 1) >='0' && temp.charAt(index+1) <='9'))
+						break;
```

Comment:
Suggesting an alternative You can also write : 
String str = input.split("itemId=")[1].replace("/", "").split("&")[0];
System.out.println("Item id is " + str);

This two line in try block and it will work.